Doc/library/binascii.rst

```diff
@@ -155,6 +155,24 @@ The :mod:`binascii` module defines the following functions:
    Similar functionality (but returning a text string) is also conveniently
    accessible using the :meth:`bytes.hex` method.
 
+   If *sep* is specified, it must be a single character str or bytes object.
+   It will be inserted in the output after every *bytes_per_sep* input bytes.
+   Separator placement is counted from the right end of the output by default,
+   if you wish to count from the left, supply a negative *bytes_per_sep* value.
+
+      >>> import binascii
```

Comment:
```suggestion
     >>> import binascii
```

Seems to be misaligned with below set of >>>.

Reply:
Can confirm locally that there is no extra space here and below docs. Could be something wrong with GitHub rendering. Sorry for the noise.